deployment/data-streams/changeset/call_configurator_set_staging_config.go

```diff
@@ -0,0 +1,172 @@
+package changeset
+
+import (
+	"errors"
+	"fmt"
+	"math/big"
+
+	"github.com/ethereum/go-ethereum/accounts/abi/bind"
+	"github.com/ethereum/go-ethereum/common"
+	ethTypes "github.com/ethereum/go-ethereum/core/types"
+
+	owner_helpers "github.com/smartcontractkit/ccip-owner-contracts/pkg/gethwrappers"
+	"github.com/smartcontractkit/ccip-owner-contracts/pkg/proposal/mcms"
+	"github.com/smartcontractkit/ccip-owner-contracts/pkg/proposal/timelock"
+
+	"github.com/smartcontractkit/chainlink/deployment"
+	"github.com/smartcontractkit/chainlink/deployment/common/changeset"
+	"github.com/smartcontractkit/chainlink/deployment/common/proposalutils"
+	"github.com/smartcontractkit/chainlink/deployment/common/types"
+	"github.com/smartcontractkit/chainlink/v2/core/gethwrappers/llo-feeds/generated/configurator"
+)
+
+type ConfiguratorConfig interface {
+	GetConfiguratorAddress() common.Address
+}
+
+type SetStagingConfigConfig struct {
+	ConfigurationsByChain map[uint64][]SetStagingConfig
+	MCMSConfig            *MCMSConfig
```

Comment:
Does every `SetConfig` type of call need this MCMS config

Reply:
I think everything that is called through MCMS needs something like that.